knn-transformers.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>Maybe add more here saying why the look reasonable i.e., the neighbors graph changes so the UMAP/clustering does as well, but the fact that both look similar indicates that the underlying graph is very close.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scanpy-tutorials/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>